### YamlMime:ManagedReference
items:
- uid: System.Data.Services.Client.DataServiceClientException
  id: DataServiceClientException
  children:
  - System.Data.Services.Client.DataServiceClientException.#ctor
  - System.Data.Services.Client.DataServiceClientException.#ctor(System.String)
  - System.Data.Services.Client.DataServiceClientException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception)
  - System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Int32)
  - System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception,System.Int32)
  - System.Data.Services.Client.DataServiceClientException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Data.Services.Client.DataServiceClientException.StatusCode
  langs:
  - csharp
  name: DataServiceClientException
  nameWithType: DataServiceClientException
  fullName: System.Data.Services.Client.DataServiceClientException
  type: Class
  summary: "A végrehajtása során előforduló hibákat jelöli [!INCLUDE[ssAstoria](~/add/includes/ssastoria-md.md)] ügyfélalkalmazások számára."
  syntax:
    content: >-
      [System.Diagnostics.DebuggerDisplay("{Message}")]

      public sealed class DataServiceClientException : InvalidOperationException
  inheritance:
  - System.InvalidOperationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor
  id: '#ctor'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException()
  nameWithType: DataServiceClientException.DataServiceClientException()
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException()
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> osztály rendszer által megadott üzenetet, amely leírja a hibát."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság egy rendszer által megadott üzenetet, amely leírja a hibát az új példányának.</xref:System.Exception.Message%2A> Ez az üzenet figyelembe veszi a rendszer jelenlegi kulturális környezethez.       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceClientException ();
    parameters: []
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException(String)
  nameWithType: DataServiceClientException.DataServiceClientException(String)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> osztály egy adott üzenet, amely leírja a hibát."
  remarks: "Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Data.Services.Client.DataServiceClientException>.</xref:System.Data.Services.Client.DataServiceClientException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A hiba üzenet mezőben megadott karakterlánc `message`. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceClientException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceClientException.DataServiceClientException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  syntax:
    content: protected DataServiceClientException (System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: serializationInfo
      type: System.Runtime.Serialization.SerializationInfo
      description: "Fel kell venni a párbeszédpanelre."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Fel kell venni a párbeszédpanelre."
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException(String,Exception)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza vagy `null` Ha a <xref:System.Exception.InnerException%2A>tulajdonság nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Data.Services.Client.DataServiceClientException>.</xref:System.Data.Services.Client.DataServiceClientException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A hiba üzenet mezőben megadott karakterlánc `message`. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceClientException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a jelenlegi kivétel a következik be egy <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokkot, amely kezeli a belső kivételben találhatók."
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Int32)
  id: '#ctor(System.String,System.Int32)'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException(String,Int32)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Int32)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Int32)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> osztály."
  syntax:
    content: public DataServiceClientException (string message, int statusCode);
    parameters:
    - id: message
      type: System.String
      description: "A karakterlánc-érték, amely tartalmazza a hibaüzenetet."
    - id: statusCode
      type: System.Int32
      description: "Az egész szám, amely tartalmazza az állapotkód."
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception,System.Int32)
  id: '#ctor(System.String,System.Exception,System.Int32)'
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: DataServiceClientException(String,Exception,Int32)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Exception,Int32)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Exception,Int32)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> osztály."
  syntax:
    content: public DataServiceClientException (string message, Exception innerException, int statusCode);
    parameters:
    - id: message
      type: System.String
      description: "A karakterlánc-érték, amely tartalmazza a hibaüzenetet."
    - id: innerException
      type: System.Exception
      description: "A <xref uid=&quot;langword_csharp_System.Exception&quot; name=&quot;System.Exception&quot; href=&quot;&quot;> </xref> objektum, amely tartalmazza a belső kivételben találhatók."
    - id: statusCode
      type: System.Int32
      description: "Az egész szám, amely tartalmazza az állapotkód."
  overload: System.Data.Services.Client.DataServiceClientException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: DataServiceClientException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceClientException.GetObjectData(SerializationInfo,StreamingContext)
  type: Method
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref> a szerializált adatok osztályt."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben. További információkért lásd: [XML és a SOAP-szerializálási](~/add/includes/ajax-current-ext-md.md)."
  syntax:
    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Data.Services.Client.DataServiceClientException.GetObjectData*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceClientException.StatusCode
  id: StatusCode
  parent: System.Data.Services.Client.DataServiceClientException
  langs:
  - csharp
  name: StatusCode
  nameWithType: DataServiceClientException.StatusCode
  fullName: System.Data.Services.Client.DataServiceClientException.StatusCode
  type: Property
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Lekérdezi a után visszaadott HTTP-állapot hibakód <xref href=&quot;System.Data.Services.Client.DataServiceClientException&quot;> </xref>."
  syntax:
    content: public int StatusCode { get; }
    return:
      type: System.Int32
      description: "A kivétel jelölő egész szám."
  overload: System.Data.Services.Client.DataServiceClientException.StatusCode*
  exceptions: []
  platform:
  - net462
references:
- uid: System.InvalidOperationException
  isExternal: false
  name: System.InvalidOperationException
- uid: System.Data.Services.Client.DataServiceClientException.#ctor
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException()
  nameWithType: DataServiceClientException.DataServiceClientException()
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException()
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException(String)
  nameWithType: DataServiceClientException.DataServiceClientException(String)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceClientException.DataServiceClientException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException(String,Exception)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Int32)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException(String,Int32)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Int32)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Data.Services.Client.DataServiceClientException.#ctor(System.String,System.Exception,System.Int32)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException(String,Exception,Int32)
  nameWithType: DataServiceClientException.DataServiceClientException(String,Exception,Int32)
  fullName: System.Data.Services.Client.DataServiceClientException.DataServiceClientException(String,Exception,Int32)
- uid: System.Data.Services.Client.DataServiceClientException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: DataServiceClientException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceClientException.GetObjectData(SerializationInfo,StreamingContext)
- uid: System.Data.Services.Client.DataServiceClientException.StatusCode
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: StatusCode
  nameWithType: DataServiceClientException.StatusCode
  fullName: System.Data.Services.Client.DataServiceClientException.StatusCode
- uid: System.Data.Services.Client.DataServiceClientException.#ctor*
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: DataServiceClientException
  nameWithType: DataServiceClientException.DataServiceClientException
- uid: System.Data.Services.Client.DataServiceClientException.GetObjectData*
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: GetObjectData
  nameWithType: DataServiceClientException.GetObjectData
- uid: System.Data.Services.Client.DataServiceClientException.StatusCode*
  parent: System.Data.Services.Client.DataServiceClientException
  isExternal: false
  name: StatusCode
  nameWithType: DataServiceClientException.StatusCode
